tools/driver/CMakeLists.txt

```diff
@@ -51,6 +51,10 @@ endif()
 
 add_dependencies(clang clang-headers)
 
+if (DEFINED CHECKEDC_IN_TREE)
```

Comment:
Can we also change this to `if(CHECKEDC_IN_TREE)` as it should always be defined with the patch I've just sent you for checkedc-llvm.